package.json

```diff
@@ -36,6 +36,8 @@
     },
     "dependencies": {
         "body-scroll-lock": "^3.1.5",
+        "chartjs-adapter-date-fns": "^2.0.0",
+        "date-fns": "^2.25.0",
```

Comment:
If I remember correctly it's because these weren't needed there, but here cause we use the dates on our price chart, will have to double check today to be sure.

Reply:
did you check in the meantime @Highjhacker

Reply:
follow up question, why was `date-fns` necessary now compared to the previous chartjs version where we didn't use it?

Reply:
I thought it would be necessary given that change in the migration guide ; 

```
moment is no longer specified as an npm dependency. If you are using the time or timeseries scales, you must include one of the available adapters and corresponding date library. You no longer need to exclude moment from your build.
```

But apparently it isn't, still using that `time` scale while having these two extra dependencies removed, and it still works fine. Gonna check a bit more but we should be able to ditch it apparently.